catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/NsiliClassificationComparator.java

```diff
@@ -25,8 +25,7 @@ public int compare(NsiliClassification o1, NsiliClassification o2) {
     if (o1 != NsiliClassification.NO_CLASSIFICATION
         && (o2 == null || o2 == NsiliClassification.NO_CLASSIFICATION)) {
       return -1;
-    } else if ((o1 == null || o1 == NsiliClassification.NO_CLASSIFICATION)
-        && o2 != NsiliClassification.NO_CLASSIFICATION) {
+    } else if (o1 == null || o1 == NsiliClassification.NO_CLASSIFICATION) {
```

Comment:
❓ Maybe I'm missing something. Can we really remove the `o2 != NsiliClassification.NO_CLASSIFICATION` check here? It seems like that's changing the boolean logic.

Reply:
Took me a while walking through that to confirm the change yields identical results, but it does.